src/renderer/views/SubscribedChannels/SubscribedChannels.vue

```diff
@@ -73,5 +73,170 @@
   </div>
 </template>
 
-<script src="./SubscribedChannels.js" />
+<script setup>
+import { computed, onMounted, onBeforeUnmount, ref, watch } from 'vue'
+import FtCard from '../../components/ft-card/ft-card.vue'
+import FtFlexBox from '../../components/ft-flex-box/ft-flex-box.vue'
+import FtInput from '../../components/ft-input/ft-input.vue'
+import FtSubscribeButton from '../../components/ft-subscribe-button/ft-subscribe-button.vue'
+import { invidiousGetChannelInfo, youtubeImageUrlToInvidious, invidiousImageUrlToInvidious } from '../../helpers/api/invidious'
+import { getLocalChannel, parseLocalChannelHeader } from '../../helpers/api/local'
+import { ctrlFHandler } from '../../helpers/utils'
+import { useI18n } from '../../composables/use-i18n-polyfill.js'
+import store from '../../store/index'
+
+const { locale } = useI18n()
+
+const re = {
+  url: /(.+=\w)\d+(.+)/,
+  ivToYt: /^.+ggpht\/(.+)/
+}
+const ytBaseURL = 'https://yt3.ggpht.com'
+
+const query = ref('')
+const subscribedChannels = ref([])
+const filteredChannels = ref([])
+const thumbnailSize = ref(176)
+const errorCount = ref(0)
```

Comment:
Looks like these don't need reactivity, thumbnailSize could be a normal const and errorCount a normal let variable.